modules/oadp-timeouts.adoc

```diff
@@ -0,0 +1,194 @@
+[id="oadp-timeouts_{context}"]
+= OADP timeouts
+
+Extending a timeout allows complex or resource-intensive processes to complete successfully without premature termination, reducing the likelihood of errors, retries or failures. However, it is necessary to strike a balance when extending timeouts. Setting excessively long timeouts may hide underlying issues in the process. Therefore, careful consideration and monitoring are necessary to determine an appropriate timeout value that meets the needs of the process and the overall system performance.
+
+Below are various OADP timeouts, with instructions of how and when to implement these parameters:
+
+[id="Restic-timeout_{context}"]
+== Restic timeout
+
+.Description
+`timeout` defines the Restic timeout. The default value is 1h.
+
+.Implementation
+.. Use this parameter for Restic backups with large volumes.
+.. For Restic backups only. Use this parameter if backups are timing out with the following error:
++
+[source,terminal]
+----
+level=error msg="Error backing up item" backup=velero/monitoring error="timed out waiting for all PodVolumeBackups to complete"
+----
+
+.Procedure
+* Edit the values in the `spec.configuration.restic.timeout` block of the `DataProtectionApplication` CR manifest, as in the following example:
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+ name: <dpa_name>
+spec:
+  configuration:
+    restic:
+      timeout: 1h
+  ...
+----
+
+[id="Velero-resource-timeout_{context}"]
+== Velereo resource timeout
+
+.Description
+`resourceTimeout` defines how long to wait for several Velero resources before timeout occurs, such as Velero CRD availability, `volumeSnapshot` deletion, and repo availability. The default is 10m.
+
+.Implementation
+.. Use this timeout for backups with very large volumes . This parameter is used as a timeout value when Velero tries to clean-up or delete the Container Storage Interface (CSI) snapshots, before marking the backup as complete.
+... A sub-task of this cleanup tries to patch VSC and this timeout can be used for that task.
++
+.. Use this timeout to create or ensure a backup repository is ready for filesystem based backups for Restic or Kopia.
+.. Use this timeout to check if the Velero CRD is available in the cluster before restoring the custom resource (CR) or resource from the backup.
+
+.Procedure
+* Edit the values in the `spec.configuration.velero.resourceTimeout` block of the `DataProtectionApplication` CR manifest, as in the following example:
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+ name: <dpa_name>
+spec:
+  configuration:
+    velero:
+      resourceTimeout: 10m
+  ...
+----
+
+[id="Datamover-timeout_{context}"]
+== Data Mover timeout
+
+.Description
+`timeout` is a user supplied timeout to be used for `VolumeSnapshotBackup` and `VolumeSnapshotRestore` to complete. The default value is 10m.
```

Comment:
`timeout` was also used in the Restic timeout listing. These should be merged or the lead-in sentences should focus more on the specific parameters, IMO, to avoid confusion.

Reply:
s/user supplied/user-supplied